package.json

```
@@ -31,6 +31,7 @@
     "test:node": "mocha node-tests --exclude node-tests/local/**",
     "test:node-local": "mocha node-tests/local",
     "test:node-local-exclude-api-urls": "FOLLOW_API_URLS=false npm run test:node-local",
+    "release:guides:minor": "scripts/create-new-version",
```

I think, when you renamed the script file, you forgot to update the line here.
```suggestion
    "release:guides:minor": "scripts/create-new-minor-version",
```

---

Thanks for catching it!